### PR TITLE
NO-JIRA: Enhance aks docs

### PR DESCRIPTION
**Pull request by fxierh**
Enhance aks docs by:
- specifying a custom --node-vm-size
- Dynamically obtain the image digests
- applying some formatting

Note: 
- the default vm size (Standard_DS2_v2) is not sufficient in terms of memory
```bash
# One hosted cluster
$ k top node
NAME CPU(cores) CPU% MEMORY(bytes) MEMORY%
aks-nodepool1-10175151-vmss000000 1728m 90% 4762Mi 111%
aks-nodepool1-10175151-vmss000001 556m 29% 1871Mi 43%
aks-nodepool1-10175151-vmss000002 1739m 91% 4815Mi 112%
```

### CODE REVIEW

File: docs/content/how-to/azure/create-azure-cluster_on_aks.md [hunk 1]
```diff
@@ -164,16 +165,31 @@ az network vnet create \
...
+## Get index image digests
+```
+HYPERSHIFT_HC_VERSION="4.16"
+
+CERTIFIED_OPERATOR_INDEX_REPO="registry.redhat.io/redhat/certified-operator-index"
+COMMUNITY_OPERATOR_INDEX_REPO="registry.redhat.io/redhat/community-operator-index"
+REDHAT_MARKETPLACE_INDEX_REPO="registry.redhat.io/redhat/redhat-marketplace-index"
+REDHAT_OPERATOR_INDEX_REPO="registry.redhat.io/redhat/redhat-operator-index"
+
+CERTIFIED_OPERATOR_INDEX_HASH="$(oc image info "${CERTIFIED_OPERATOR_INDEX_REPO}:v${HYPERSHIFT_HC_VERSION}" -a "${PULL_SECRET}" --filter-by-os linux/amd64 -o json | jq -r .listDigest)"
```
Comment (bryan-cox) [inline docs/content/how-to/azure/create-azure-cluster_on_aks.md:187]: Is there a way to pull multi-arch versions of these? What I previously had listed are multi-arch versions.

Comment (fxierh) [reply docs/content/how-to/azure/create-azure-cluster_on_aks.md:187]: I think these images are multi-arched:
```bash
fxie-mac:hypershift fxie$ oc image info "registry.redhat.io/redhat/redhat-marketplace-index:v4.16" -a .dockerconfigjson --filter-by-os linux/amd64 -o json | jq -r .listDigest
sha256:7a3c1318682731d00ffa023ec6aa9de6ccb847510a4e419a3f2f5303b8599a21

fxie-mac:hypershift fxie$ podman pull registry.redhat.io/redhat/redhat-marketplace-index@sha256:7a3c1318682731d00ffa023ec6aa9de6ccb847510a4e419a3f2f5303b8599a21
Trying to pull registry.redhat.io/redhat/redhat-marketplace-index@sha256:7a3c1318682731d00ffa023ec6aa9de6ccb847510a4e419a3f2f5303b8599a21...
Getting image source signatures
Checking if image destination supports signatures
Copying blob sha256:952362883a155a409d36ac8d3b94589d90e3712df70ac9311eac571669853b92
Copying blob sha256:ca3fb8a8b0720e3d4111993f744300d47111b7915db490dfc85bde84b47def66
Copying blob sha256:7082f02cc356245ebfcfeae2c489bb614e7fa6e96a45ef9ab92e41a75ccaadaf
Copying blob sha256:15003cddfdb3e3c76299eac8c8da4bcbeb28d3f52029143a3d9c803edd7016a5
Copying blob sha256:eeef8bb22bccb1e6a72c43715e4cb9b776d52e2acd09417c12e8cccdcb4c288c
Copying config sha256:f68296b312db2d1baa9db5e15e9acfeaf87eb78e40ac684297f8c2ad343daa40
Writing manifest to image destination
Storing signatures
f68296b312db2d1baa9db5e15e9acfeaf87eb78e40ac684297f8c2ad343daa40

fxie-mac:hypershift fxie$ podman manifest inspect registry.redhat.io/redhat/redhat-marketplace-index@sha256:7a3c1318682731d00ffa023ec6aa9de6ccb847510a4e419a3f2f5303b8599a21
{
    "schemaVersion": 2,
    "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
    "manifests": [
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 1091,
            "digest": "sha256:b9771bf4c68f4a09cc6552df11fb77d88685795134a055a91995101db0574905",
            "platform": {
                "architecture": "amd64",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 1091,
            "digest": "sha256:8818bf7d94fa4ff4f302d5f5dcaafd18be642fb29af04ed276404e216d763029",
            "platform": {
                "architecture": "arm64",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 1091,
            "digest": "sha256:6f7550db8a55939b5f27bcf643b67018ed63bf278dda142b9237ebd8431a20cc",
            "platform": {
                "architecture": "ppc64le",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 1091,
            "digest": "sha256:eb41496cdc51e51e90c74a0f2fa79d2d9002a3a463245f1d6a878d08972a5f6f",
            "platform": {
                "architecture": "s390x",
                "os": "linux"
            }
        }
    ]
}
```

**Pull request merged**